Intersect (Core)/Config/CombatOptions.cs

```diff
@@ -17,6 +17,8 @@ public class CombatOptions
         //Combat
         public int RegenTime = 3000; //3 seconds
 
+        public bool CombatRegen = false; // Enable or disable regeneration in combat.
```

Comment:
Instead of making this a comment could you please make it actual XML documentation?

Reply:
Last night I considered doing it but it was 1AM, today I will adjust the comment correctly.